commands/store_inspect.go

```diff
@@ -32,31 +32,35 @@ func runStoreInspect(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("please provide the function name")
 	}
 
+	targetPlatform := getTargetPlatform(platformValue)
 	storeItems, err := storeList(storeAddress)
 	if err != nil {
 		return err
 	}
 
-	item := storeFindFunction(args[0], storeItems)
+	platformFunctions := filterStoreList(storeItems, targetPlatform)
+
+	functionName := args[0]
```

Comment:
Can we do something for the times I directly write `ASCII Cows` without wrapping it in `''`? My motivation is primary UX here and I don't see `faas-cli store inspect` being used in different context/having different usage other than inspecting specific function. We can get all the arguments after `faas-cli store inspect` and cat them and then compare values.

Reply:
Martin that is a good suggestion. Can you raise that as a separate issue?

@viveksyngh can you answer whether you addressed this or not? (I'm not saying that you have to, I just want to know what was done)

Reply:
I also think this is very good suggestion somehow I have lost this comment. It is not addressed in this PR.

Reply:
I will review it later today and will open Issue about this

Reply:
@martindekov following up on this from June, can you take a look today?

Reply:
Sorry for the delay with the answer Alex, I had [Issue](https://github.com/openfaas/faas-cli/issues/657) opened, but me and @LucasRoesler resolved it as unnecessary 👍